### PR TITLE
Added example site at top level of source tree

### PR DESCRIPTION
**Pull request by runarbe**
Two new files:
index.html and styles.css - this will be the entry point for learning about s4a.js and how it may be used

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/*
+To change this license header, choose License Headers in Project Properties.
+To change this template file, choose Tools | Templates
+and open the template in the editor.
+*/
+/* 
+    Created on : 01.des.2015, 07:35:54
+    Author     : runarbe
```
Comment (eirikcsak) [inline styles.css]: Author runarbe?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            </div>
+        </div>
+
+        <div class="container">
+            <!-- Example row of columns -->
+            <div class="row">
+                <div class="col-md-4">
+                    <h2>License</h2>
+                    <p>the library is released by the SDI4Apps consortium under an Apache 2.0 license</p>
+                    <p><a class="btn btn-default" href="LICENSE" role="button">View license &raquo;</a></p>
```
Comment (eirikcsak) [inline index.html:70]: File extension "LICENSE" are not standard og will by default be blocked by IIS and probably apache

Comment (runarbe) [inline index.html:70]: I do not think it a good idea to duplicate LICENSE information into another file, setting the configuration for IIS to serve extension-less files as text/html is easy:

```
<staticContent>
   <mimeMap fileExtension="." mimeType="text/html" />
</staticContent>
```

**Comment by runarbe** [comment]
@eirikcsak Thanks for the inputs, review again and merge if ok

**Pull request closed without merging**